test/unit/test_config.py

```diff
@@ -840,6 +841,7 @@ def test_nested_none_values(self):
         self.assertEqual(project.models, {'vars': {}, 'pre-hook': [], 'post-hook': []})
         self.assertEqual(project.seeds, {'vars': {}, 'pre-hook': [], 'post-hook': [], 'column_types': {}})
 
+    @pytest.mark.skipif(os.name == 'nt', reason='crashes CI for Windows')
```

Comment:
I tuned off this test on Windows and created #3233 to figure out what is going on with it!